Solutions/Island/Data Connectors/IslandAdminAPIConnector.json

```diff
@@ -75,14 +75,14 @@
             "instructionSteps": [
               {
                 "title": "Connect Island to Microsoft Sentinel",
-                "description": "Provide the Island API Key.",
+                "description": "Provide the Island API URL and Key.  API URL is https://management.island.io/api/external/v1/adminActions for US or https://eu.management.island.io/api/external/v1/adminActions for EU.\n  Generate the API Key in the Management Console under Settings > API.",
```

Comment:
we should probably add UK as well

Reply:
not needed today and will resolve long term with doc reference since list of api urls will continue to grow and become unwieldy in this interface.